gtest/src/js/mod.rs

```diff
@@ -0,0 +1,176 @@
+// This file is part of Gear.
+
+// Copyright (C) 2021 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use log::debug;
+use std::path::Path;
+use std::process::Command;
+
+#[allow(dead_code)]
+pub enum MetaType {
+    InitInput,
+    InitOutput,
+    Input,
+    Output,
+}
+
+impl ToString for MetaType {
+    fn to_string(&self) -> String {
+        match self {
+            MetaType::InitInput => "init_input",
+            MetaType::InitOutput => "init_output",
+            MetaType::Input => "input",
+            MetaType::Output => "output",
+        }
+        .into()
+    }
+}
+
+pub fn gear_path() -> String {
+    let pwd = Command::new("pwd")
+        .output()
+        .expect("Unable to call pwd command");
+
+    let path = String::from_utf8(pwd.stdout).expect("Unable to parse pwd output bytes");
+    let path_parts: Vec<String> = path.split("/").map(|v| v.replace("\n", "")).collect();
+
+    if let Some(index) = path_parts.iter().rposition(|r| r == "gear") {
+        path_parts[..index + 1].join("/")
+    } else {
+        panic!("Gear root directory not found")
+    }
+}
+
+pub fn get_bytes(path: &str, meta_type: MetaType, json: String) -> Vec<u8> {
+    let mut wasm_path = gear_path();
+    wasm_path.push_str("/");
+    wasm_path.push_str(path);
+
+    if !wasm_path.ends_with(".meta.wasm") {
+        wasm_path = wasm_path.replace(".wasm", ".meta.wasm");
+    }
+
+    if !Path::new(&wasm_path).exists() {
+        panic!("Could not find file {}", wasm_path);
+    }
+
+    let mut script_path = gear_path();
+    script_path.push_str("/gtest/src/js/encode.js");
+
+    let output = Command::new("node")
+        .arg(script_path)
+        .args(&["-p", &wasm_path, "-t", &meta_type.to_string(), "-j", &json])
+        .output()
+        .expect("Unable to call node.js process");
+
+    debug!(
```

Comment:
Use the same convection as for the functions.
```suggestion
    log::debug!(
```